skills/semantic_diff.md

```
@@ -0,0 +1,323 @@
+      Old: function bar(): Promise<User>
+      New: function bar(): Promise<User | null>
+    action: require_null_handling
+```
+
+---
+
+## Dependency Impact Visualization
+
+### Call Graph Analysis
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)